pkg/apis/config/feature_flags.go

```diff
@@ -223,6 +236,22 @@ func setEmbeddedStatus(cfgMap map[string]string, defaultValue string, feature *s
 	return nil
 }
 
+// setCustomTaskVersion sets the "custom-task-version" flag based on the content of a given map.
+// If the feature gate is invalid or missing then an error is returned.
+func setCustomTaskVersion(cfgMap map[string]string, defaultValue string, feature *string) error {
+	value := defaultValue
+	if cfg, ok := cfgMap[customTaskVersion]; ok {
+		value = strings.ToLower(cfg)
+	}
+	switch value {
+	case CustomTaskVersionAlpha, CustomTaskVersionBeta:
+		*feature = value
+	default:
+		return fmt.Errorf("invalid value for feature flag %q: %q", customTaskVersion, value)
```

Comment:
let's add a test for an invalid value in `TestNewFeatureFlagsConfigMapErrors` https://github.com/tektoncd/pipeline/blob/f62563fadf809f5def179a8f40ae3dcfc6316c7f/pkg/apis/config/feature_flags_test.go#L193-L212